inlang/source-code/website/package.json

```diff
@@ -53,7 +51,6 @@
 		"flat": "^5.0.2",
 		"isomorphic-git": "1.24.5",
 		"jose": "^4.11.1",
-		"markdown-to-txt": "^2.0.1",
 		"marked": "^4.2.4",
 		"mermaid": "^9.3.0",
```

Comment:
Those two dependencies seem markdown service related. Try to remove them and see if everything still works.